lib/ansible/module_utils/vmware.py

```diff
@@ -213,6 +213,19 @@ def find_host_portgroup_by_name(host, portgroup_name):
     return None
 
 
+def _get_vm_prop(vm, attributes):
+    """Safely get a property or return None"""
+    result = vm
+    for attribute in attributes:
+        try:
+            result = getattr(result, attribute)
+        except AttributeError:
+            return None
+        except IndexError:
+            return None
```

Comment:
Could try 

``` python
def _get_vm_prop(vm, *attributes):
    result = vm
    for attribute in attributes:
    < etc >
```

That version is a tiny bit magic, but it would allow:

``` python
_get_vm_prop(vm, 'some_attr')

# as well as
_get_vm_prop(vm, 'some_attr', 'another_attr', 'etc_attr')
```